src/pages/resultsView/coExpression/CoExpressionTab.tsx

```diff
@@ -90,55 +134,120 @@ export default class CoExpressionTab extends React.Component<ICoExpressionTabPro
         );
     }
 
+    @computed get xOptions() {
```

Comment:
I think xOptions is too vague a name - what kind of options are they? `xProfileOptions` would be better